draft-ietf-acme-dns-account-01.mkd

```diff
@@ -120,52 +216,56 @@ A client can fulfill this challenge by performing the following steps:
 
 - Construct a key authorization {{!RFC8555, Section 8.1}} from the `token` value provided in the challenge and the client's account key
 - Compute the SHA-256 digest {{FIPS180-4}} of the key authorization
-- Construct the validation domain name by prepending the following label to the domain name being validated:
+- Construct the authorization domain name by prepending the following label to the domain name being validated:
 
-      "_acme-challenge_" || base32(SHA-256(Account Resource URL)[0:9])
+      "_" || base32(SHA-256(<ACCOUNT_RESOURCE_URL>)[0:9]) || "._acme-" || <SCOPE> ||"-challenge"
 
    - SHA-256 is the SHA hashing operation defined in {{!RFC6234}}
-   - `[0:9]` is the operation that selects the first ten bytes (bytes 0 through 9 inclusive) from the previous SHA256 operation
+   - `[0:9]` is the operation that selects the first ten bytes (bytes 0 through 9 inclusive) from the previous SHA-256 operation
```

Comment:
Is there prior art for using `[0:9]` as _inclusive_ indexing in RFCs? I think just about every programming language and pseudocode uses left-exclusive indexing, and so it would be clearer to use `[0:10]` here.

Reply:
I wanted to use the syntax for open & closed intervals: https://en.wikipedia.org/wiki/Interval_(mathematics)

But that doesn't make sense since it's not applicable to discrete math anyway. I'll switch to `[0:9]`